### DISCUSSION TITLE
Why My Website Page Doesn't Appear Correctly in GitHub pages

### DISCUSSION THREAD

**Discussion by TarekHassan1** [community/community#45737] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i just learned Git command and tried to push my website docs to github and then i published it through github pages
but it didn't work saw a video on youtube and still doesn't work tried to upload files directly instead of using git 
and it still didn't Work i cant find What is the problem here 
here's The link https://tarekhassan1.github.io/HTML-CSS-Template/
it only Says " HTML-CSS-Template" 
No html no css no images nothing

**Comment by TarekHassan1** [comment, marked as answer]
actually i solved it it so simple i was just searching the wrong thing 
just Make Your Main Html File as the main file 
or just do this https://YourUserName.github.io/{Repo-Name}/{Main-Html-File}.html

**Reply by Baraka24** [reply]
I did so also but not working

**Reply by radar-blip** [reply]
@TarekHassan1,
If you rename `Template1.html` to `index.html`
your site should work as expected.

**Comment by Baraka24** [comment]
The name of your repository must be the same as your  github username. See my website https://baraka24.github.io/
